### PR TITLE
Adding data-placement attribute to popovers so they aren't clipped within scrollable panels

### PR DESCRIPTION
**Pull request by sg00dwin**
Fixes https://github.com/openshift/origin/issues/7723

### CODE REVIEW

**Comment by sg00dwin** [comment]
@spadgett for review

File: assets/app/views/directives/_warnings-popover.html [hunk 1]
```diff
@@ -2,9 +2,10 @@
   <span
     dynamic-content="{{content}}"
     data-toggle="popover"
-    data-placement="right"
+    data-placement="auto"
```
Comment (spadgett) [inline assets/app/views/directives/_warnings-popover.html]: Does `"auto right"` place it to the right when there is room?

**Comment by spadgett** [comment]
If every popover should use `data-container="body"`, we might add that to popover.js. Why is that needed, though?

File: assets/app/views/directives/_warnings-popover.html (hunk 1 above)
Comment (sg00dwin) [inline assets/app/views/directives/_warnings-popover.html]: Yes it should.

**Comment by spadgett** [comment]
Try

```
      link: function($scope, element, attrs) {
        var popupConfig = {
          container: "body"
        };
```

and

```
       $(element).popover(popupConfig);
```

in popups.js for both popovers and tooltips.

**Comment by jwforres** [comment]
assuming we are changing this to be in the directive JS instead of changing it in all the views then i think this should be in 1.2

**Comment by spadgett** [comment]
@jwforres it's not just appending to the body unfortunately, it's tooltip placement, too. @sg00dwin was going to add placement to popups.js, but we'd still want to remove data-placement attributes from the views.

If we think this should go in post 1.2, we could look at things like uib-popover.

https://angular-ui.github.io/bootstrap/#/popover

**Comment by jwforres** [comment]
on the fence, i'd probably be ok with 1.2 if we are able to test all the
ones we change

**Comment by sg00dwin** [comment]
Closing and replaced with https://github.com/openshift/origin/pull/7897

**Pull request closed without merging**